CalibMuon/DTCalibration/python/dtT0WireCalibration_cfg.py

```diff
@@ -7,6 +7,10 @@
 process.MessageLogger.destinations = cms.untracked.vstring('cerr')
```

Comment:
@saghosh if running on top of CMSSW_10_6_X_2019-04-01-1100 this cfg fails:

```
----- Begin Fatal Exception 01-Apr-2019 17:42:55 CEST-----------------------
An exception of category 'NoRecord' occurred while
   [0] Processing global begin Run run: 312774
   [1] Calling method for module DTDigiTask/'dtTPmonitor'
Exception Message:
No "MuonGeometryRecord" record found in the EventSetup.n
 The Record is delivered by an ESSource or ESProducer but there is no valid IOV for the synchronizatio value.
 Please check 
   a) if the synchronization value is reasonable and report to the hypernews if it is not.
   b) else check that all ESSources have been properly configured. 
----- End Fatal Exception -------------------------------------------------
```

Did you observe this in your tests?

Reply:
@saghosh Global tag is missing in the config file. And it doesn't look like GT will be configured by some external parameter either.
You may consider using
process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')

Reply:
@fabiocos @tocheng We tested this code as a part of our workflow for the calibration, and we did not see the error messages, mainly because this code is not used as in by running something like cmsRun, but actually the calibration workflow uses this file as a template and also adds the dataset and globalTag information in subsequent steps before the creating a final config file based on the user input that runs without any problems.
Also, please note, this file is not meant to be run on collisions data but is used for the T0 calibration that uses special test pulse runs. I hope this clarifies your comments, please let me know if the clarification is not satisfactory...